<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Netlist Hierarchy Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTreeWidget" name="treeWidget">
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="indentation">
         <number>10</number>
        </property>
        <property name="animated">
         <bool>false</bool>
        </property>
        <column>
         <property name="text">
          <string>Items</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTreeWidget" name="propertyTree">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <attribute name="headerDefaultSectionSize">
       <number>200</number>
      </attribute>
      <column>
       <property name="text">
        <string>Property</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Value</string>
       </property>
      </column>
      <item>
       <property name="text">
        <string>hide_name</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>attributes</string>
       </property>
       <item>
        <property name="text">
         <string>src</string>
        </property>
        <property name="text">
         <string>input.v:17</string>
        </property>
       </item>
      </item>
      <item>
       <property name="text">
        <string>parameters</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <item>
        <property name="text">
         <string>A_SIGNED</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>A_WIDTH</string>
        </property>
        <property name="text">
         <string>9</string>
        </property>
       </item>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="resource_tree_map_btn">
   <property name="text">
    <string>Resource Tree Map</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
